hathor/event/event_manager.py

```diff
@@ -36,7 +36,6 @@
     HathorEvents.NETWORK_NEW_TX_ACCEPTED,
     HathorEvents.NETWORK_BEST_BLOCK_FOUND,
     HathorEvents.NETWORK_ORPHAN_BLOCK_FOUND,
-    HathorEvents.LOAD_STARTED,
```

Comment:
Why won't we emit `LOAD_STARTED` as well?

Reply:
I removed this event since it was supposed to be emitted in the next line of code where the `MANAGER_ON_START` event is emitted, and therefore it looked like an unnecessary duplication.

It could indeed be necessary if we want it to carry some information, but that is currently not foreseen in the design. About full verification, I discussed this with @jansegre and we are not supporting simultaneously enabling the events feature and full verification flags, since this would create problems and new edge cases to be handled during the load phase. This is enforced in this PR. Also, in that subject, please read https://github.com/HathorNetwork/hathor-core/pull/534#discussion_r1128974258.

Reply:
Got it. Ok, we can remove it for now.